pkgs/applications/radio/uhd/default.nix

```diff
@@ -13,7 +13,7 @@
 , python3
 , buildPackages
 , enableExamples ? false
-, enableUtils ? false
+, enableUtils ? true
 , libusb1
 , enableDpdk ? false
```

Comment:
Could we enable this feature by default while at it? Same for `enablePythonApi` - if we include Python references, let's go all the way.